### PR TITLE
Runner outputing logits or distribution

### PR DESCRIPTION
**Pull request by jlibovicky**
This PR introduces a runner that produces logits or a normalized distribution from a decoder (i.e., currently a recurrent decoder or a classifier) unlike the greedy runner which produces the argmax of the distributions.

### CODE REVIEW

File: neuralmonkey/runners/logits_runner.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+# pylint: disable=too-few-public-methods
+class LogitsRunner(BaseRunner):
+    """A runner which takes the output from decoder.decoded."""
+
+    def __init__(self,
+                 output_series: str,
+                 decoder: Any,
+                 normalize: bool = True,
+                 pick_index: int = None) -> None:
```
Comment (jindrahelcl) [inline neuralmonkey/runners/logits_runner.py]: přidal bych docstring, kterej vysvětlí tyhle argumenty

File: neuralmonkey/runners/logits_runner.py [hunk 2]
```diff
@@ -68,17 +68,49 @@ def collect_results(self, results: List[Dict]) -> None:
...
+        """Initializes the logits runner.
+
+        Args:
+            output_series: Name of the series produces by the runner.
+            decoder: A decoder having logits.
+            normalize: Flag whether the logits should be normalized with
+                softmax.
+            pick_index: If not None, it specifies the index of the logit or the
+                probability that should be on output.
+            pick_value: If not None, it specifies a value from the decoder's
```
Comment (jindrahelcl) [inline neuralmonkey/runners/logits_runner.py]: hezká featura
